test/unittests/t_util.cc

```diff
@@ -424,7 +425,8 @@ TEST_F(T_Util, MakePipe) {
   void *buffer_output = scalloc(100, sizeof(char));
   MakePipe(fd);
   write(fd[1], to_write.c_str(), to_write.length());
-  read(fd[0], buffer_output, to_write.length());
+  int bytes_read = read(fd[0], buffer_output, to_write.length());
+  EXPECT_EQ(bytes_read, to_write.length());
```

Comment:
Yes, you are right. I think it should now be fixed. In any case for the tests we run there shouldn't be a problem because we deal with very small values, always positive.